server/services/notifications/caAndDmNotificationSender.js

```diff
@@ -112,28 +112,19 @@ module.exports = function createCaAndDmNotificationSender(
       return []
     }
 
-    return mailboxes.map(mailbox => {
+    return mailboxes.map((mailbox) => {
       return { personalisation: { ...common, dm_name: mailbox.name }, email: mailbox.email, templateName: 'STANDARD' }
     })
   }
 
   const notificationDataMethod = {
     CA_RETURN: R.compose(
-      R.then(replaceTemplateNames({ STANDARD: 'CA_RETURN', COPY: 'CA_RETURN_COPY' })),
+      R.andThen(replaceTemplateNames({ STANDARD: 'CA_RETURN', COPY: 'CA_RETURN_COPY' })),
```

Comment:
changes to ramda api - this clashed with the concept of thenables so they had to rename the method.